internal/clients/readiness.go

```diff
@@ -15,7 +14,7 @@ import (
 )
 
 const (
-	readinessCheckTimeout = time.Second
+	readinessCheckTimeout = 5 * time.Second
```

Comment:
Given we execute all the checks in parallel, it makes sense to increase the timeout. 👍